cylc/uiserver/schema.py

```diff
@@ -697,35 +714,31 @@ def run_jobs_query(
             task_jobs
         '''
     if where_stmts:
-        query += 'WHERE\n            ' + '\n            AND '.join(where_stmts)
+        query += 'WHERE ' + ' AND '.join(where_stmts)
```

Comment:
The newlines and indenting made debugging this much easier.

Reply:
But makes the code more difficult to read. If we want to make debugging easier we could/should dedent the whole query in the first place?